panels/site_sections/attractions_admin.py

```diff
@@ -5,10 +5,11 @@
 
 @all_renderable(c.STUFF)
 class Root:
+    @renderable_override(c.STUFF, c.PEOPLE, c.REG_AT_CON)
     def index(self, session, filtered=False, message='', **params):
+        admin_account = session.current_admin_account()
         if filtered:
-            admin_account_id = cherrypy.session['account_id']
-            attraction_filter = [Attraction.owner_id == admin_account_id]
+            attraction_filter = [Attraction.owner_id == admin_account.id]
```

Comment:
I can't figure out why Github is showing the above two lines as a change.  It looks like the exact same line with no changes is showing up as removed and re-added.  Am I missing something?

Reply:
`admin_account_id` vs `admin_account.id`

Note the underscore (`_id`) vs the period (`.id`).

Reply:
Ahhh, I figured it was something like that but I stared at that line for like 30 seconds and still didn't see it.  Derp.